src/shared/proto/tbots_timestamp_msg.proto

```diff
@@ -0,0 +1,7 @@
+syntax = "proto3";
+
+message TimestampMsg
+{
+    // Total seconds of UTC time since Unix epoch
+    double epoch_timestamp = 1;
```

Comment:
we have timestamps in milliseconds and seconds it would be good to still keep the unit as `seconds` imo

Reply:
yup it's still seconds but with more precision

Reply:
right I meant doing `epoch_timestamp_seconds` to be super explicit